data/856/826/15/85682615.geojson

```diff
@@ -2,417 +2,241 @@
-    "ne:iso_3166_2":"ES-",
-    "ne:iso_a2":"ES",
-    "ne:labelrank":"3",
-    "ne:latitude":"42.89020000000",
-    "ne:longitude":"-2.69660000000",
-    "ne:mapcolor13":5,
-    "ne:mapcolor9":5,
-    "ne:name":"\u00c1lava",
-    "ne:name_alt":"",
-    "ne:name_len":5,
-    "ne:name_local":"",
-    "ne:note":"",
-    "ne:postal":"PV",
-    "ne:provnum_ne":"18",
-    "ne:region":"Pa\u00eds Vasco",
-    "ne:region_cod":"ES.PV",
-    "ne:region_sub":"",
-    "ne:sameascity":"-99",
-    "ne:scalerank":"3",
-    "ne:sov_a3":"ESP",
-    "ne:sub_code":"",
-    "ne:type":"Comunidad Aut\u00f3noma",
-    "ne:type_en":"Autonomous Community",
-    "ne:wikipedia":"",
-    "ne:woe_id":12602134,
-    "ne:woe_label":"Basque Country, ES, Spain",
-    "ne:woe_name":"Pa\u00eds Vasco",
-    "qs:a0":"Spain",
-    "qs:a1":"Araba/\u00c1lava",
-    "qs:a1_lc":"01",
-    "qs:adm0":"Spain",
-    "qs:adm0_a3":"ESP",
-    "qs:level":"adm1",
-    "qs:source":"Spain IGN",
-    "src:geom":"esp-cnig",
-    "src:geom_alt":[
-        "quattroshapes",
-        "whosonfirst"
-    ],
-    "src:geom_via":"quattroshapes",
-    "src:lbl_centroid":"mapshaper",
-    "src:population":"eurostat",
-    "src:population_date":"2022",
-    "unlc:subdivision":"ES-VI",
-    "wof:belongsto":[
-        102191581,
-        85633129,
-        404227393
-    ],
-    "wof:breaches":[],
-    "wof:concordances":{
-        "digitalenvoy:region_code":11468,
-        "esp-cnig:inspireid":"ES.IGN.BDDAE.34160100000",
-        "esp-ine:code":"01",
-        "eurostat:nuts_2021_id":"ES211",
-        "fips:code":"SP94",
-        "gn:id":3130717,
-        "gp:id":12602134,
-        "hasc:id":"ES.PV",
-        "iso:code":"ES-VI",
-        "unlc:id":"ES-VI"
+    "edtf:cessation": "",
+    "edtf:inception": "",
+    "esp-cnig:connut1": "ES2",
+    "esp-cnig:connut2": "ES2",
+    "esp-cnig:inspireid": "ES.IGN.BDDAE.34160100000",
+    "esp-cnig:nameunit": "Araba/Álava",
+    "eurostat:coast_type": "3",
+    "eurostat:country_code": "ES",
+    "eurostat:level_code": "3",
+    "eurostat:mount_type": "3",
+    "eurostat:name_latin": "Araba/Álava",
+    "eurostat:name_local": "Araba/Álava",
+    "eurostat:nuts_2021_id": "ES211",
+    "eurostat:nuts_2021_level_0": "ES",
+    "eurostat:nuts_2021_level_1": "ES2",
+    "eurostat:nuts_2021_level_2": "ES21",
+    "eurostat:nuts_2021_level_3": "ES211",
+    "eurostat:population": 329798,
+    "eurostat:population_year": 2022,
+    "eurostat:urban_type": "1",
+    "geom:area": 0.334436040891571,
+    "geom:area_square_m": 3032512884.396035,
+    "geom:bbox": "-3.286767,42.472361,-2.232687,43.217187",
+    "geom:latitude": 42.83503110460992,
+    "geom:longitude": -2.7205292079752623,
+    "iso:country": "ES",
+    "label:eng_x_preferred_longname": [
+      "Alava Province"
+    ],
+    "label:eng_x_preferred_placetype": [
+      "province"
+    ],
+    "label:eng_x_preferred_shortcode": [
+      "PV"
+    ],
+    "label:spa_x_preferred_longname": [
+      "Provincia de Álava"
```

Comment:
@thisisaaronland How are we handling Á accent marks, has that changed with the Go tools?